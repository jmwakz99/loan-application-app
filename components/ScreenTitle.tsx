import { FC } from "react";
import { Text } from "react-native";

import { styles } from "@/styles/screenTitle";

type Props = {
  title: string;
};

const ScreenTitle: FC<Props> = ({ title }) => {
  return <Text style={styles.screenTitleText}>{title}</Text>;
};

export default ScreenTitle;
