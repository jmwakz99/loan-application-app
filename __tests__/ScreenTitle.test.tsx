import { render } from "@testing-library/react-native";

import ScreenTitle from "@/components/ScreenTitle";

describe("ScreenTitle", () => {
  it("It renders corectly", () => {
    const title = "Test Title";

    const { getByText } = render(<ScreenTitle title={title} />);

    const screenTitle = getByText("Test Title");

    expect(screenTitle).toBeTruthy();
  });

  it("snapshot", () => {
    const title = "Test Title";

    const screenTitle = render(<ScreenTitle title={title} />).toJSON();

    expect(screenTitle).toMatchSnapshot();
  });
});
