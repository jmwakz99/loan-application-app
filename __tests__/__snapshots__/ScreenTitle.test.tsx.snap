// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders corectly 1`] = `
<Text
  style={
    {
      "color": "#000000",
      "fontFamily": "Roboto-Bold",
      "fontSize": 64,
      "fontWeight": "800",
      "letterSpacing": 1,
    }
  }
>
  Test Title
</Text>
`;
